### DISCUSSION TITLE
comando

### DISCUSSION THREAD

**Discussion by guiworldtv2** [LITUATUI/M3UPT#13] [Q&A]
conheces algum comando e alguma forma automatizada a atualizar os links da lista m3u?
consigo alguns links de alguns canais de tv direto da fonte mas eles se inspiram após algumas horas, sabes de alguma forma a burlar isso?

**Comment by LITUATUI** [comment]
Podes ver este [script](https://github.com/LITUATUI/M3UPT/blob/main/streams.sh
) que serve para actualizar o URL temporário dos streams dos canais TVI.

É uma adaptação [deste](https://github.com/marmelo/streaming-pt/blob/master/tv.sh).
